config.go

```diff
@@ -442,7 +443,7 @@ type Config struct {
 	// the copy will likely be.
 	//
 	// Optional: defaults to 200
-	DataIterationBatchSize uint64
```

Comment:
We should keep this so we can specify a default value, and the `map` will be an override. Otherwise if you have a lot of tables, you would have to specify this number for every table.